src/i18n.js

```diff
@@ -0,0 +1,21 @@
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+// NOTE: this is temporary. uncomment when in dev
+// import translations from './core/i18n/translations';
+
+i18n
+  .use(initReactI18next)
+  .use(Backend)
+  .use(LanguageDetector)
+  .init({
+    lng: 'en',
+    backend: {
+      loadPath: 'static/i18n/{{lng}}/{{ns}}.json'
+      // loadPath: (language) => translations[language]
```

Comment:
As far as I know, this will not work in production. It needs to remain as a function.
```suggestion
      // loadPath: 'static/i18n/{{lng}}/{{ns}}.json'
      loadPath: (language) => translations[language]
```

Reply:
related to https://github.com/openlattice/chronicle-web/pull/584#discussion_r607255053

Reply:
The build outputs a filename like this:

```
static/i18n/sv/translation.7a6c6159a43ff4632b1dc8588a8151ba.json
```

I'm not sure if `{{lng}}/{{ns}}.json` will work, we can test it. However, the `translation` object will contain the correct string value which is why it's implemented with `translations[language]`.